README.md

```diff
@@ -58,6 +58,77 @@ By default you can login with any non-empty username and password.
 
 [php-web-server]: https://www.php.net/manual/en/features.commandline.webserver.php
 
+### Running with Docker
```

Comment:
```suggestion
### Developing with Docker
```
I really don't want to give the idea that the IDE should be being run under Docker.